guides/plugins/plugins/plugin-fundamentals/programmically-sending-emails.md

```diff
@@ -0,0 +1,119 @@
+# Programmically sending Emails
+
+In this guide you'll learn how to send Emails programmically using the `Mailservice` provided in Shopware v6.4.x.
+
+## Prerequisites
+
+In order to add the MailService for your plugin, you first need a plugin as base. Therefore, you can refer to the [Plugin Base Guide](../plugin-base-guide.md) and [how to add a custom controller](./add-custom-controller)
```

Comment:
In order to use the mail service.

- either refer to it as mail service or if you use the actual class name, put it in back ticks: `MailService`.

Reply:
I use the latter one, thanks :)